package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ksyos/react-dropzone-component",
   "description": "A Dropzone Component for ReactJS",
-  "version": "3.2.1",
```

Comment:
I'm not quite sure how the versioning is handled in this repository. Should this be `3.2.2` after the package upgrade?

Reply:
Maybe it could, since it's forked already. I simply thought it was a typo back in the day, just because there was a jump from `3.1.2` to `3.2.1`, but in this case I don't think there are any changes that are worth a release (the original repo keeps the `3.2.1` release from 2018). Can @ccoenraad have a word on this, maybe?

Reply:
This repo hasn't been updated in over two years so I'm not sure what the plan is here. @remco-zwartepoorte is it still used (actively) anywhere or is there a plan to deprecate this package?

Reply:
This repo is indeed actively being used in the `AssetDropzone` component. Until we have build a completely new Dropzone component, this package is still needed. I agree with @claudio-lc that this package upgrade PR is not worth a release.